src/commissaire/containermgr/trivial.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2017  Red Hat, Inc
```

Comment:
I'll have to generate a custom container-manager config file for the non-standard module path, but I think so.  (We generate one already for the storage service.)